### DISCUSSION TITLE
Trouble on GitHub Global Campus application process

### DISCUSSION THREAD

**Discussion by LizhengyuSJTU** [community/community#138236] [GitHub Education]
### Hi everyone,

Dear GitHub Education Team,
    I must apologize for my being unable to provide paper academic affiliation documents or be in the campus, but it is the vacation duration and I cannot have access to any paper proof. However I can let the office of academic affairs of my university to send an email to confirm my status. I wonder if that could be an alterative.
    Best Wishes
LS

**Comment by alx-mp4** [comment]
Hi @LizhengyuSJTU, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
